server/sqlflowserver.go

```diff
@@ -55,34 +56,60 @@ func (s *Server) Run(req *pb.Request, stream pb.SQLFlow_RunServer) error {
 		}
 		defer db.Close()
 	}
-	var pr *sf.PipeReader
-	if s.enableSession == true {
-		pr = s.run(req.Sql, db, s.modelDir, req.Session)
-	} else {
-		pr = s.run(req.Sql, db, s.modelDir, nil)
+
+	// FIXME(typhoonzero): split by ; can not deal with situations like
+	// "SELECT * from mytable where col <> ';';", should be fixed.
```

Comment:
I support to allow SQLFlow server to run a sequence of SQL statements to form a workflow. But I have a few questions:

1. I cannot find a design doc. However, I think we do need one, and it should explain the interactions between Jupyter Notebook server/magic command and the SQLFlow server.
   - What would happen if the user presses Ctrl-Enter in a Jupyter Notebook cell with one or more SQL statements? Will there be one or more gRPC call(s)?
   - What would happen if the user clicks the Run button (a triangle pointing to the right)? Will there be one or more gRPC call(s)?

- I am very worried that the splitting of SQL statements in a cell by `;` is too hacky and would lead to many bugs. I would do the right things in the right order -- get the general parser (connecting SQLFlow with various database systems) done before parsing multiple statements from a cell.